run_brer/run_config.py

```diff
@@ -377,8 +377,12 @@ def __train(self, tpr_file=None, **kwargs):
             # of alpha.
             # noinspection PyUnresolvedReferences
             current_alpha = context.potentials[i].alpha
+            if current_alpha == 0.0:
+                raise Exception('Alpha value was constrained to 0.0')
+
             # noinspection PyUnresolvedReferences
             current_target = context.potentials[i].target
+            current_converged = getattr(context.potentials[i], 'converged', False)
```

Comment:
You still aren't doing anything with `current_converged`. If you aren't going to do anything with the value in this function, you should remove this line. If you want to log it, though, that would be fine, but see comments elsewhere about defaulting to `None` instead of `False`, and checking for `None` before using.